dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+cmake_minimum_required(VERSION 2.8)
+
```

Comment:
Can you give this project a name? Perhaps `project(OpenSimDependencies)`? Otherwise, it's just named "Project".